reference/shipping.v3.yml

```diff
@@ -397,6 +458,24 @@ components:
           readOnly: true
           format: date-time
           example: 2022-09-21T14:15:00+00:00
+    shippingSettings:
+      title: shippingSettings
+      description: Shipping settings.
+      type: object
+      properties:
+        checkout:
+          $ref: '#/components/schemas/countryListStrategy'
```

Comment:
I'm also unsure about this - is this actually `checkoutSettings`, one property of which (for now) is `country_list_strategy`?

That is, if we add another property to this area (e.g., our out of zone delivery message) it would appear as another property within `checkoutSettings`?

Reply:
@payaalaee Could you put some screenshots of the intended request and response bodies in the PR description? The other option is to enable the feature on my store 

Currently, this is how the GET response body is documented (not sure if it's right)
![GET response](https://github.com/user-attachments/assets/b360a6ac-232c-4f90-88bf-5aaf284e6909)

This is how the POST request and response is documented: 
![POST request and response](https://github.com/user-attachments/assets/c71b5c90-8820-49c7-8e9a-290ac6e09c1d)

Reply:
Thanks Andrea, Shipping Settings looks good then, thanks for confirming how the tags are used.

Curiously, do you know if a tag's `description` is used (if we define one)?

Reply:
To clarify, I understand the outcome is as closed to the expected, my suggestion is renaming the `ref`. The scenario of adding another field to the `checkout` object above highlights the incorrect abstraction.

Reply:
Thanks, the entire code in this part was wrong and I've updated it similar to the other usages. There is now only one `shippingSettings` schema with the correct format being used across get and post request and response samples. I believe this is looking good now. Thanks for your comments. I'm also attaching a screenshot of the API request and response.

Reply:
Wonderful @payaalaee, thank you!